### PR TITLE
Added verification to the input stream

### PR DESCRIPTION
**Pull request by whitesoundcl**
Checking if the file is in good condition.

### CODE REVIEW

File: onehand.cpp [hunk 1]
```diff
@@ -226,6 +226,13 @@ int main(const int argc, char* const* argv)
 		// Get input stream for file
 		std::fstream kFileStream;
 		kFileStream.open(keymapFile);
+		
+		if(!kFileStream.good())
+		{
+			fprintf(stderr, "The file is not in good condition.\n");
+			kFileStream.close();
+			return -1;
```
Comment (veryjos) [inline onehand.cpp]: Change to `exit(1)`, just for consistency.

**Review by veryjos — CHANGES_REQUESTED** [review]

**Comment by veryjos** [comment]
Looks good, just one small change and I'll merge it :~)

Thanks for pointing this out, grabbing EVIOCTL is certainly dangerous :)

**Comment by whitesoundcl** [comment]
Sorry for closing it :(

**Comment by veryjos** [comment]
Merged, thanks! :dancer:

**Pull request merged**